docs/release/release-team.md

```diff
@@ -21,7 +21,7 @@ This document introduces the concept of a release team with the following goals
 
 Note that this document is intended to be a starting point for the release team. It is not a complete release process document.
 
-More details on the CAPI release process can be found in [this past issue tracking release tasks](https://github.com/kubernetes-sigs/cluster-api/issues/6615) and the current [release documentation](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/developer/releasing.md).
+More details on the CAPI release process can be found in [this past issue tracking release tasks](https://github.com/kubernetes-sigs/cluster-api/issues/6615) and the current [release cycle](./release-cycle.md) and [release task](./release-tasks.md) documentation.
```

Comment:
should we remove the past issue reference now that we have an actual doc for tasks?

Reply:
Good point, done